src/platform/unix/linux/mod.rs

```diff
@@ -26,18 +26,64 @@ use std::thread;
 
 const MAX_FDS_IN_CMSG: u32 = 64;
 
+const SCM_RIGHTS: c_int = 0x01;
```

Comment:
It apears as if everything from `cmsg` is not defined in `libc`. It definitely seems like they should be defined somewhere like `libc` or `nix`.